src/alfasim_score/converter/alfacase/score_input_reader.py

```diff
@@ -312,3 +313,35 @@ def read_operation_fluid_data(self) -> Dict[str, Any]:
             "api_gravity": Scalar(fluid_data["api_gravity"], FRACTION_UNIT),
             "gas_gravity": Scalar(fluid_data["gas_gravity"], FRACTION_UNIT),
         }
+
+    def read_output_curves(self) -> Dict[str, Array]:
```

Comment:
Include a comment in both methods, that they are only use to "test", compare results quem presented in json.

Reply:
Ok, added the comment